src/google-classroom-extractor/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "google-classroom-extractor"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
Please remove unused (I think) libraries:

* jupyter
* matplotlib
* numpy
* seaborn
* toolz

Also remove the myget reference, as we'll never use MyGet for this project.

Reply:
Done.  Also removed last Jupyter notebook.